libs/utils/logWebVitals.js

```diff
@@ -87,9 +87,21 @@ function logMepExperiments(lanaData, mep) {
   });
 }
 
-export default function webVitals(mep, { delay = 1000 } = {}) {
-  const lanaData = {};
-  logMepExperiments(lanaData, mep);
-  observeCLS(lanaData);
-  observeLCP(lanaData, delay);
+export default function webVitals(mep, { delay = 1000, sampleRate = 50 } = {}) {
+  const isChrome = () => {
+    const nav = window.navigator;
+    return nav.userAgent.includes('Chrome') && nav.vendor.includes('Google');
+  };
+  if (!isChrome() || Math.random() * 100 > sampleRate) return;
+
+  function handleEvent() {
+    const performanceConsent = window.adobePrivacy.activeCookieGroups().indexOf('C0002') !== -1;
+    if (!performanceConsent) return;
```

Comment:
Similar comment to my previous one, why define functions and event listeners when they might not be used? I'd suggest returning early
```suggestion
  const performanceConsent = window.adobePrivacy.activeCookieGroups().indexOf('C0002') !== -1;
  if (!performanceConsent) return;

  function handleEvent() {
    const lanaData = {};
    // ...
```

Reply:
`adobePrivacy` object will be made available later in the waterfall, while the needs to execute sooner. I think the approach is correct.

Reply:
I did notice we load this quite late, so @overmyheadandbody has a point in the sense of - maybe privacy is already available in which case we would have missed the events.
I added support for this case as well